watchfaces/007-bold-hour-bebas.qml

```diff
@@ -66,7 +66,10 @@ Item {
         horizontalAlignment: Text.AlignHCenter
         anchors.horizontalCenter: parent.horizontalCenter
         anchors.verticalCenter: parent.verticalCenter
-        text: wallClock.time.toLocaleString(Qt.locale(), "HH")
+        text: if (use12H.value) {
```

Comment:
Mh, this is a lot of duplicated code :/ What about exposing a function in asteroid-launcher along the line of:

```
function paddedHour(time) {
  if (use12H.value)
    return time.toLocaleString(Qt.locale(), "hh ap").slice(0, 2)
  else
    return time.toLocaleString(Qt.locale(), "HH")
}
```

And then use it as `paddedHour(wallClock.time)` when only the time is shown or in more complex cases something like: `"<b>" + paddedHour(wallClock.time) + "</b>"  + wallClock.time.toLocaleString(Qt.locale(), ":mm"`

I think this would make the code more readable and a bit easier to maintain

Reply:
Good Idea!
I was about to have a function in every watchface but discarded that idea since it would even add more code lines for the function syntax.
How to go about? Will you implement the function so i can rework the watchfaces and update the PR?

Reply:
Ah that's a good question indeed, let me build you a modified asteroid-launcher and if you can get it to work then I push the change and then you can push your PRs :)

Reply:
so I started to implement the function in asteroid-launcher but realized it feels really out of place there so I'll just merge the PR ;) sorry for the delay and thank you for the contrib!